scrapy/settings/default_settings.py

```diff
@@ -45,7 +45,6 @@
 
 CONCURRENT_REQUESTS = 16
 CONCURRENT_REQUESTS_PER_DOMAIN = 8
-CONCURRENT_REQUESTS_PER_IP = 0
```

Comment:
I think we should keep it here during the deprecation period.

We could use a `__getattr__` module-level function to log a deprecation warning upon import. I wonder if that means we also need to define `__all__` with the whole list of settings defined here, but to be honest, it may be better to break any `from scrapy.settings.default_settings import *`, seems like a bad practice and I cannot picture a valid reason to use that.

Reply:
Ah, you mean getting the value from importing it in the user code via `from scrapy.settings.default_settings import *`. I'm fine with breaking that.

Reply:
> I'm fine with breaking that.

While I am OK with breaking it for this specific setting, I would not make it a general rule. For example, checking `scrapy.settings.default_settings.TWISTED_REACTOR` seems like a nice way to check what the default reactor is in the installed version of Scrapy without specifically checking for the Scrapy version.

Reply:
I added `__alll__` and logic to emit deprecation message when someone try to import CONCURRENT_REQUESTS_PER_IP directly.